<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NStopWatch</class>
 <widget class="QWidget" name="NStopWatch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>36</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="pushButtonStart">
     <property name="text">
      <string>Start</string>
     </property>
     <property name="icon">
      <iconset resource="../Resources.qrc">
       <normaloff>:/icons/play</normaloff>:/icons/play</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonStop">
     <property name="text">
      <string>Pause</string>
     </property>
     <property name="icon">
      <iconset resource="../Resources.qrc">
       <normaloff>:/icons/pause</normaloff>:/icons/pause</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonReset">
     <property name="text">
      <string>reset</string>
     </property>
     <property name="icon">
      <iconset resource="../Resources.qrc">
       <normaloff>:/icons/reset</normaloff>:/icons/reset</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="timeEdit">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="displayFormat">
      <string>HH:mm:ss</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonStart</sender>
   <signal>clicked()</signal>
   <receiver>NStopWatch</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStop</sender>
   <signal>clicked()</signal>
   <receiver>NStopWatch</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonReset</sender>
   <signal>clicked()</signal>
   <receiver>NStopWatch</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>start()</slot>
  <slot>stop()</slot>
  <slot>reset()</slot>
 </slots>
</ui>
